src/fastaudio/augment/signal.py

```diff
@@ -115,7 +119,7 @@ def encodes(self, sg: AudioSpectrogram):
         return shift_signal(sg, int(s), self.roll)
 
 
-class NoiseColor(Enum):
+class NoiseColor:
```

Comment:
We probably want this to be a IntEnum ([source](https://docs.python.org/3/library/enum.html#intenum))